src/global/types.h

```diff
@@ -1593,6 +1594,10 @@ typedef struct TEXTSTRING {
         int32_t h;
         int32_t v;
     } scale;
+    struct {
+        RGBA8888 centre;
+        RGBA8888 edge;
+    } centre_v_gradient;
```

Comment:
What does the `v` in `centre_v_gradient`  stand for?

Reply:
Center vertical gradient. If you have a better name lmk. I wasn't sure what to call it.

Reply:
Maybe something related to `highlight background`?